docs/google/fetch-logs.md

```diff
@@ -0,0 +1,66 @@
+# Table of Contents
+
+ - [API and server behaviours](#api-and-server-behaviours)
+ - [Self-hosted Tools](#self-hosted-tools)
+ - [Online services](#online-services)
+
+This document describes how to read data from the Certificate Transparency
+logs.
+
+# API and server behaviours
+Certificate Transparency logs make the entries available through the
+`get-entries` endpoint, as defined in
+[RFC6962](https://www.rfc-editor.org/rfc/rfc6962#section-4.6).
+
+When fetching entries from a log, always keep in mind that:
+
+ 1. `Logs MAY restrict the number of entries that can be retrieved per
+ "get-entries" request`: this means that if you make a
+ `get-entries?start=10&end=40`, request the server might return a smaller slice,
+ such as `start=10&end=20`. The number of entries returned might vary between
+ requests, so always check how many entries each response contains, to
+ dynamically adjust the start index of the next reques should be. In Trillian
```

Comment:
PTAL, I've reformulated this in a way that should be more useful for clients. I'm linking to the server code instead of the flags, such that people who are curious of how the logic works can have a look at it.